python_modules/dagster/dagster/_generate/templates/PROJECT_NAME_PLACEHOLDER/README.md

```diff
@@ -4,7 +4,7 @@ This is a [Dagster](https://dagster.io/) project scaffolded with [`dagster proje
 
 ## Getting started
 
-First, install your Dagster repository as a Python package. By using the --editable flag, pip will install your repository in ["editable mode"](https://pip.pypa.io/en/latest/topics/local-project-installs/#editable-installs) so that as you develop, local code changes will automatically apply.
+First, install your Dagster code location as a Python package. By using the --editable flag, pip will install your code location in ["editable mode"](https://pip.pypa.io/en/latest/topics/local-project-installs/#editable-installs) so that as you develop, local code changes will automatically apply.
```

Comment:
I think it is important that when we refer something that `pip` is acting on we use the term Python module.

e.g. By using the --editable flag, pip will install your ******. 

Right now it is code location but I think we should say "Python package" or "Python project".